app/src/marketplace/containers/EditProductPage/BeneficiaryAddress.jsx

```diff
@@ -32,16 +34,26 @@ const BeneficiaryAddress = ({ address, onChange, disabled, className }: Props) =
                     <Translate value="editProductPage.setPrice.setRecipientEthAddress" />
                 </strong>
                 <div>
-                    <TextField
+                    <Text
                         id="beneficiaryAddress"
                         autoComplete="off"
-                        value={address || ''}
+                        defaultValue={address || ''}
```

Comment:
`defaultValue` does nothing if `address` changes that's why "Fill wallet address" won't work. I do it better in #832. Plus, this may work just fine for `ProductName` components, etc. (ones where the value relies only on the user input).